packages/b/blake3/xmake.lua

```diff
@@ -28,6 +33,12 @@ package("blake3")
                     else
                         add_files("c/*x86-64_unix.S")
                     end
+                elseif is_arch("x86", "i386", "i686") then
```

Comment:
we need not i686, it is also i386 for mingw platform.

Reply:
Right, I wasn't sure, I'll correct this with my next commit.